src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -579,10 +571,12 @@
 		</type>
 		<type fullname="System.Reflection.ParameterInfo" preserve="fields" />
 		<!-- reflection.c: ves_icall_get_parameter_info -->
-		<type fullname="System.Reflection.RuntimeParameterInfo" >
+		<type fullname="System.Reflection.ParameterInfo" >
```

Comment:
there's already an entry for `System.Reflection.ParameterInfo` right above, should these be joined?

Reply:
Thanks - good catch. I basically removed the second one, since the first one already preserves all fields.